metaci/testresults/robot_importer.py

```diff
@@ -319,8 +319,9 @@ def export_robot_test_results(flowtask, test_results) -> None:
     payload = {
         "build": {
             "name": flowtask.build_flow.build.plan.name,
+            "org": flowtask.build_flow.build.org.name,
             "number": flowtask.id,
-            "url": "https://www.example.com/",  # flowtask.build_flow.build.get_external_url(),  # returns 'detail': [{'loc': ['body', 'build', 'url'], 'msg': 'URL host invalid, top level domain required', 'type': 'value_error.url.host'}] when sent to gus-bus,
+            "url": f"{flowtask.build_flow.build.get_external_url()}",  # if SITE_URL is set and still returning a 422 try setting an additional 127.0.0.1 domain in you /etc/hosts file
```

Comment:
Specifically, one that includes more than one part since gus-bus' validation is looking for a tld